setup.py

```diff
@@ -4,8 +4,8 @@
 from os.path import expanduser, exists
 
 # check Python version, before we do anything
-if sys.version_info[:2] not in [(2, 7), (3, 5), (3, 6)]:
-    sys.stderr.write("The Synapse Client for Python requires Python 2.7 or 3.5 or 3.6.\n")
+if sys.version_info[:2] not in [(2, 7), (3, 5), (3, 6), (3, 7)]:
+    sys.stderr.write("The Synapse Client for Python requires Python 2.7, 3.5, 3.6, or 3.7.\n")
```

Comment:
Please also make sure the supported languages also show up in PyPi (https://pypi.org/project/synapseclient/)